egov-indexer/mukta-contractservices-indexer.yml

```
@@ -0,0 +1,183 @@
+  mappings:
+    - topic: mukta-save-contract
+      configKey: INDEX
+      indexes:
+        - name: contract-inbox
+          type: general
+          id: $.id,$.tenantId
+          jsonPath: $.contract
+          timeStampField: $.auditDetails.createdTime
+          isBulk: false
+          customJsonMapping:
+            indexMapping: { "Data":{"id":"","tenantId":"","contractStatus":"","contractNumber":"","executingAuthority":"","contractType":"","totalContractedAmount":"","securityDeposit":"","agreementDate":"","issueDate":"","defectLiabilityPeriod":"","orgId":"","startDate":"","endDate":"","completionPeriod":"","status":"","lineItems":[],"@timestamp":"timestamp","auditDetails":{},"additionalDetails":{},"currentProcessInstance":{},"history":{}}}
```

yamllint

[warning] too few spaces after comma

(commas)